src/SI_Toolkit/Predictors/predictor_ODE_tf.py

```diff
@@ -31,7 +31,7 @@ def convert_to_tensors(s, Q):
 class predictor_ODE_tf(template_predictor):
     supported_computation_libraries = {TensorFlowLibrary}  # Overwrites default from parent
     
-    def __init__(self, horizon=None, dt=0.02, intermediate_steps=10, disable_individual_compilation=False, batch_size=1, planning_environment=None, **kwargs):
+    def __init__(self, horizon: int, dt: float, intermediate_steps=10, disable_individual_compilation=False, batch_size=1, **kwargs):
```

Comment:
Made these arguments non-optional.

@marcinpaluch1994 is there a case where we can pass no `horizon` or `dt` and expect it to work?